openfisca_france/model/prestations/aide_permis_demandeur_emploi.py

```diff
@@ -0,0 +1,59 @@
+from openfisca_france.model.base import Variable, Individu, TypesActivite, MONTH
+
+
+class aide_permis_demandeur_emploi_eligibilite_financiere(Variable):
+    value_type = bool
+    entity = Individu
+    label = "Éligibilité financière à l'aide à l’obtention du permis de conduire automobile pour les demandeurs d’emploi"
+    reference = [
+        "Délibération Pôle Emploi n°2011/13 du 11 avril 2011",
+        "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n201113-du-11-avril.html?type=dossiers/2011/bope-n2011-36-du-18-avril-2011",
+        "Bulletin officiel de Pôle emploi n°2020-05 du 17 janvier 2020",
+        "http://www.bo-pole-emploi.org/bulletinsofficiels/instruction-pe-n2011-205-du-9-decembre-2011--mise-a-jour-bope-n2020-05.html?type=dossiers/2020/bope-n2020-005-du-17-janvier-2020",
+        ]
+    definition_period = MONTH
+
+    def formula(individu, period, parameters):
+        sans_rsa = individu.famille('rsa', period) <= 0
+        sans_aah = individu('aah', period) <= 0
+        sans_retraite = individu('retraite_brute', period) <= 0
+
+        allocation_journaliere_minimum = parameters(period).allocation_retour_emploi.montant_minimum
+        plafond_chomage = allocation_journaliere_minimum * 31
+        chomage_minimum = individu('chomage_brut', period) <= plafond_chomage
```

Comment:
> indemnisé au titre d’une allocation chômage dont le montant **net** (avant prélèvement à la source de l’impôt sur le revenu) est inférieur ou égal à l’allocation d’aide au retour à l’emploi minimale (ARE minimale )

[Source : 1.1.1](http://www.bo-pole-emploi.org/bulletinsofficiels/instruction-pe-n2011-205-du-9-decembre-2011--mise-a-jour-bope-n2020-05.html?type=dossiers/2020/bope-n2020-005-du-17-janvier-2020)

```suggestion
        chomage_minimum = individu('chomage_net', period) <= plafond_chomage
```